content/en/docs/installation/_index.en.md

```diff
@@ -139,14 +139,10 @@ helm install --version {{% release-version %}} --namespace $FISSION_NAMESPACE fi
 
 This method uses `kubectl apply` to install Fission.
 You can edit the YAML file before applying it to your cluster, if you want to change anything in it.
-Create namespace for fission installation.
 
-```sh
-kubectl create namespace fission
-```
 
 {{% notice info %}}
-If you want to install in another namespace, please consider to use `helm` or generate yaml for first with `helm template` command.
+Fission is the recommended namespace to install fissionIf you want to install in another namespace, please consider to use `helm` or generate yaml for first with `helm template` command.
```

Comment:
`fission` is the recommended namespace to install fission.

`fission` Please format namespace name
fullstop missing after sentense

Reply:
formated fission and added full stop